vignettes/estimate_infections_options.Rmd

```diff
@@ -254,26 +262,26 @@ dep <- estimate_infections(reported_cases,
     pop = 1000000, future = "latest"
   )
 )
-#> Warning: There were 1 divergent transitions after warmup. See
+#> Warning: There were 16 divergent transitions after warmup. See
```

Comment:
Ah this is not ideal. So this suggest that alpha isn't well identified here? I'm open to either of these suggestions (also in combination with #841).

Reply:
Yes I think it is very hard to identify both this and the length scale. Looking at pairs plots before this definitely seemed to be where the divergent transitions were coming from. That being said g said it could lead to overly estimates in some cases (I.e the motivation for changing)

Reply:
Is there any case for tuning the prior of the length scale? Having read across some stan references I think there might be a case for having this a bit tighter (as indeed we seem to already have when `ls_sd=0` as implemented in https://github.com/epiforecasts/EpiNow2/pull/257).

Reply:
I mean we can do but I think its a choice between tuning alpha of the lengthscale